src/sentry/integrations/slack/utils.py

```diff
@@ -172,12 +174,19 @@ def build_group_attachment(group, event=None, tags=None, identity=None, actions=
 
     ignore_button = {"name": "status", "value": "ignored", "type": "button", "text": "Ignore"}
 
-    has_releases = Release.objects.filter(
-        projects=group.project, organization_id=group.project.organization_id
-    ).exists()
-
-    if not has_releases:
-        resolve_button.update({"name": "status", "text": "Resolve", "value": "resolved"})
+    project = Project.objects.get_from_cache(id=project_id)
+
+    cache_key = "has_releases:1:%s" % hash_values([project.id, project.organization_id])
+    has_releases = cache.get(cache_key)
+    if has_releases is None:
+        has_releases = Release.objects.filter(
+            projects=project, organization_id=project.organization_id
+        ).exists()
+        if has_releases:
+            cache.set(cache_key, has_releases, 3600)
+        else:
+            cache.set(cache_key, 0, 60)
```

Comment:
Let me know if I should use a different value here. I'm unsure about the semantics of Null vs Nullish values in the cache.

Reply:
Should the endpoints that create releases clear this key?

Reply:
We rarely explicitly delete keys, setting it for a minute is low enough for it to clear on itself, imo.